packages/teleterm/src/ui/AppInitializer.tsx

```diff
@@ -1,18 +1,23 @@
-import React, { ReactNode, useEffect } from 'react';
+import React, { FC, useEffect } from 'react';
 import { useAppContext } from 'teleterm/ui/appContextProvider';
+import useAsync from 'teleterm/ui/useAsync';
+import styled from 'styled-components';
 
-interface AppInitializerProps {
-  children?: ReactNode;
-}
-
-export function AppInitializer(props: AppInitializerProps) {
+export const AppInitializer: FC = props => {
```

Comment:
What does moving this logic here gives us? From what I can tell, the loading message appears later than before.

Reply:
IMO there was no benefit of doing it in `boot.tsx` (maybe it was a bit faster, but I think it can be counted in milliseconds). By moving it to the `AppInitializer` we have correct styling for `Loading` out of the box, provided by global style from `styled-components` (and in `boot.tsx` we couldn't even use it). 
Additionally, in your PR https://github.com/gravitational/webapps/pull/713/files you had to manually import `theme`.  It has a drawback of not being responsive to changes, so when we would add a white theme we would have to import another file (but we will still have to handle it for `BrowserWindow.backgroundColor`).